content/posts/weekly/August-2022.md

```diff
@@ -0,0 +1,11 @@
+---
+title: August 2022
+date: 2022-08-01
+weight: 59
+---
+
+## Week 30 (July 26th–August 1st)
+
+- Packit can now build RPMs in mock. For more information see https://packit.dev/docs/cli/build/mock ([packit#1662](https://github.com/packit/packit/pull/1662))
+- When using Packit before being allowed, Packit newly links an approval issue where the self-approval can be performed. ([packit-service#1596](https://github.com/packit/packit-service/pull/1596))
+- A downstream koji-build can now be re-triggered by adding a PR comment containing `/packit koji-build`. ([packit-service#1586](https://github.com/packit/packit-service/pull/1586))
```

Comment:
```suggestion
- A downstream koji-build can now be re-triggered by adding a PR comment containing `/packit koji-build` into a dist-git pull request with target branch corresponding to the branch the build should be acted upon. ([packit-service#1586](https://github.com/packit/packit-service/pull/1586))
```
Or something similar to be explicit. Also, @majamassarini you have recently tested this, is it already working and can we announce this?

Reply:
I have not yet seen a re-triggered koji-build in stage but I hope to have solved all the issues. I plan to re-test it with the next ogr or packit release. Maybe we can wait one ore week before advertising it?